hack/generate-manifest-windows.sh

```diff
@@ -22,9 +22,12 @@ function echoerr {
 
 _usage="Usage: $0 [--mode (dev|release)] [--keep] [--help|-h]
 Generate a YAML manifest to run Antrea on Windows Nodes, using Kustomize, and print it to stdout.
-        --mode (dev|release)  Choose the configuration variant that you need (default is 'dev')
-        --keep                Debug flag which will preserve the generated kustomization.yml
-        --help, -h            Print this message and exit
+        --mode (dev|release)            Choose the configuration variant that you need (default is 'dev')
+        --keep                          Debug flag which will preserve the generated kustomization.yml
+        --help, -h                      Print this message and exit
+        --containerd                    Support for containerd runtime. 
+        --ovs-run-mode (pod|service)    Run Windows OVS processes inside a container in antrea-agent pod or
```

Comment:
Please update comments to explain `ovs-run-mode` can't be used with docker runtime.